.changeset/happy-actors-cover.md

```diff
@@ -0,0 +1,31 @@
+---
+'@jpmorganchase/mosaic-plugins': patch
+'@jpmorganchase/mosaic-cli': patch
+'@jpmorganchase/mosaic-components': patch
+'@jpmorganchase/mosaic-labs-components': patch
+'@jpmorganchase/mosaic-content-editor-plugin': patch
+'@jpmorganchase/mosaic-core': patch
+'@jpmorganchase/mosaic-create-site': patch
+'@jpmorganchase/mosaic-from-http-request': patch
+'@jpmorganchase/mosaic-layouts': patch
+'@jpmorganchase/mosaic-open-api-component': patch
+'@jpmorganchase/mosaic-schemas': patch
+'@jpmorganchase/mosaic-serialisers': patch
+'@jpmorganchase/mosaic-site': patch
+'@jpmorganchase/mosaic-site-components': patch
+'@jpmorganchase/mosaic-site-middleware': patch
+'@jpmorganchase/mosaic-site-preset-styles': patch
+'@jpmorganchase/mosaic-source-git-repo': patch
+'@jpmorganchase/mosaic-source-http': patch
+'@jpmorganchase/mosaic-source-local-folder': patch
+'@jpmorganchase/mosaic-standard-generator': patch
+'@jpmorganchase/mosaic-store': patch
+'@jpmorganchase/mosaic-theme': patch
+'@jpmorganchase/mosaic-types': patch
+'@jpmorganchase/mosaic-workflows': patch
+---
+
+1. Pip Salt version
```

Comment:
Tip: imagine reading the changelog as someone who is not in the team and not familiar with the change

Changelogs are documentation and often viewed in retrospect, so handy to add as much info as possible.